docs/contributing/coding-style.md

```diff
@@ -558,6 +558,73 @@ func New(opts ...Option) (Server, error) {
 }
 ```
 
+### Constructor
+
+In Vald, the functional option pattern is widely used when we create an object.
+
+When setting the value with the functional option, the value is validated inside the option method.
+
+However, if we forget to set the required fields when creating the object, it may not be detected.
+
+Therefore, we strongly suggest to validate the object at the end of initialization.
```

Comment:
```suggestion
Therefore, we strongly suggest to validate the object during initialization.
```

Reply:
Thank you for your comment!
I fixed it!